src/shareon.js

```diff
@@ -85,6 +85,12 @@ const init = () => {
               via: encodeURIComponent(
                 child.dataset.via || container.dataset.via || ""
               ),
+              hashtag: encodeURIComponent(
+                child.dataset.hashtag || container.dataset.hashtag || ""
+              ),
+              hashtags: encodeURIComponent(
+                child.dataset.hashtags?.replaceAll(', ', ',') || container.dataset.hashtags?.replaceAll(', ', ',') || ""
```

Comment:
Not a fan of this extra data validation. To keep the bundle size small, let's not fix the website owners' errors

```suggestion
                child.dataset.hashtags || container.dataset.hashtags || ""
```